style.css

```diff
@@ -342,3 +342,151 @@ body > * > * {
     fill: white;
   }
 }
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: space-between;
+}
+
+.github-buttons-container {
+  display: flex;
+  flex-direction: row;
+  text-shadow: none;
+}
+
+.github-buttons-container > div:not(:first-child) {
+  margin-left: 0.5rem;
+}
+
+/* From https://github.com/mdo/github-buttons/blob/master/src/styles.css */
```

Comment:
@guimcaballero I think here we can link to the original project, as you've done.
Then add something along these lines:

```
/* Licensed under Apache 2.0 <link to their license file> */
/* No modifications were made from the original source */
```

Reply:
Done!